_rules/SC1-1-1-accessible-name-is-image-filename.md

```diff
@@ -0,0 +1,95 @@
+---
+name: Image accessible name is filename
+description: |
+  This rule checks that image elements that use their source filename as their accessible name do so without loss of infomation to the user.
+
+success_criterion:
+- 1.1.1 # Non-Text Content
+
+test_aspects:
+- DOM Tree
+- CSS Styling
+
+authors:
+- Bryn Anderson
+---
+
+## Test procedure
+
+### Applicability
+
+The rule applies to any HTML `input` element with a [`type`](https://www.w3.org/TR/html/sec-forms.html#dom-htmlinputelement-type) of `image`, or any HTML `area` element, or any HTML element with the semantic role of `img`, where the element is [non-decorative](#decorative) and is [included in the accessibility tree](#included-in-the-accessibility-tree) and has an [accessible name](#accessible-name) that is equivalent to the [filename](#filename) specified in the `src` attribute. Difference in letter casing, and forward and trailing whitespace should be ignored. 
+
+### Expectation
+
+Each test target has an accessible name that serves only an equivalent purpose of the [non-text content](https://www.w3.org/TR/WCAG21/#dfn-non-text-content).
```

Comment:
@annethyme you had concerns about this being inconsistent with the way the ARIA rules are formulated. Do we need to look at switching this around?